p2panda-encryption/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! `p2panda-encryption` provides decentralized, secure data- and message encryption for groups
+//! with post-compromise security and optional forward secrecy.
+//!
+//! The crate implements two different group key-agreement and encryption schemes for a whole range
+//! of use cases for applications which can't rely on a stable network connection or centralised
+//! coordination.
+//!
+//! The first scheme we simply call [**"Data Encryption"**](data_scheme), allowing peers to encrypt any data with
+//! a secret, symmetric key for a group (using XChaCha20-Poly1305). This will be useful for building
+//! applications where users who enter a group late will still have access to previously created
```

Comment:
I think this is another place where I made some suggested changes in the `README` which could be copy-pasted into this section.